.babel-preset.js

```diff
@@ -1,7 +1,7 @@
 const r = m => require.resolve(m)
 
 function preset(context, options = {}) {
-  const { browser = false, debug = false } = options
+  const { browser = false, debug = false, modules = 'commonjs' } = options
```

Comment:
the output of the transpilation during the rollup build needs to preserve the es-module syntax so that rollup can properly do the rest of its processing. the default retains the previous behavior, but this allows overriding how the modules are handled.